tests/phpunit/integration/Core/Admin/DashboardTest.php

```diff
@@ -20,16 +21,173 @@
  */
 class DashboardTest extends TestCase {
 
+	/**
+	 * Admin user ID.
+	 *
+	 * @var int
+	 */
+	private $admin_id;
+
+	private $context;
+
+	/**
+	 * Set up the test.
+	 */
+	public function setUp(): void {
+		parent::setUp();
```

Comment:
Please use `set_up()` for consistency with the other test files.

```suggestion
	public function set_up(): void {
		parent::set_up();
```